Salwador/Views/ContentView.swift

```diff
@@ -10,12 +10,7 @@ import UIKit
 
 struct ContentView: View {
     @State private var prompText: String = ""
-    @State private var buttonText: String = "Ready?"
-    @State private var imagePath: String = "salvador-man"
-    @State private var image: UIImage? = nil
-    @State private var isLoading: Bool = false
-    @State private var hasError: Bool = false
-    @State private var errorMsg: String = ""
+    @ObservedObject var iGViewModel = IGViewModel()
```

Comment:
There is also the property wrapper `@StateObject`. `@StateObject` and `@ObservedObject` do very similar things but it's good to know that both exist and when to use what. Just wanted to make you aware of it. 

Here a nice article about the difference:
https://www.avanderlee.com/swiftui/stateobject-observedobject-differences/

Furthermore I would just call the view model `... var viewModel ...`